packages/components/psammead-figure/src/index.stories.jsx

```diff
@@ -27,20 +30,23 @@ storiesOf('Figure', module)
   )
   .add(
     'containing Image, ImagePlaceholder, Copyright and Caption',
-    () => (
-      <Figure>
-        <ImagePlaceholder ratio={imageRatio}>
-          <Image alt={imageAlt} src={imageSrc} width={imageWidth} />
-          <Copyright>
-            <VisuallyHiddenText>Image copyright, </VisuallyHiddenText>
-            Copyright
-          </Copyright>
-        </ImagePlaceholder>
-        <Caption>
-          <VisuallyHiddenText>Image caption, </VisuallyHiddenText>
-          Caption
-        </Caption>
-      </Figure>
+    inputProvider(
+      ['copyright offscreen text', 'caption', 'caption offscreen text'],
+      (copyrightOffscreen, caption, captionOffscreen) => (
+        <Figure>
+          <ImagePlaceholder ratio={imageRatio}>
+            <Image alt={imageAlt} src={imageSrc} width={imageWidth} />
+            <Copyright>
+              <VisuallyHiddenText>{copyrightOffscreen}</VisuallyHiddenText>
```

Comment:
Can these have defaults, where theyre what they were before?

Reply:
Not without first rewriting `inputProvider` to do that

Reply:
Visually hidden text defaults set back